helm/g8s-oauth2-proxy-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: oauth2-proxy
+  name: oauth2-proxy
+  namespace: kube-system
+spec:
+  replicas: 1
```

Comment:
can this not run with 2?

Reply:
Sure! Secrets are already deployed https://github.com/giantswarm/installations/pull/381. Deploying service to installations with this: https://github.com/giantswarm/installations/pull/382